test/config.js

```diff
@@ -2,6 +2,11 @@
 const ConfigurationBase = require('mongodb-test-runner').ConfigurationBase;
 const f = require('util').format;
 
+const chai = require('chai');
```

Comment:
Is there a reason we are adding these things in?

Reply:
I was actually glad to have found this, we often have cases where we `expect(err).to.not.exist`, however if it _does_ it shows something like `Expected { Object(namespace, message) } to be null` which is not helpful _at all_. The truncation configuration option here expands to show the full message. The other options here are just the defaults, but I'm otherwise codifying that we are indeed using the defaults.